contracts/base/Escrow.sol

```diff
@@ -169,13 +169,18 @@ contract Escrow is EscrowInterface, TInitializable, BaseUpgradeable, BaseEscrowD
     function repay(uint256 amount) external onlyOwner {
         IERC20 token = IERC20(loans.lendingToken());
         uint256 balance = _balanceOf(loans.lendingToken());
```

Comment:
Replace loans.lendingToken() with token (or lendingToken if previous suggestion is applied)